lib/rules/indent.js

```diff
@@ -809,6 +809,10 @@ module.exports = {
             },
 
             MemberExpression: function(node) {
+                if (options.MemberExpression === null) {
```

Comment:
If I do that, I have to do it like this:

    if (!context.options[1] || !("MemberExpression" in context.options[1])) {

Any reason not to go with this simpler formulation?:

    if (options.MemberExpression === undefined) {

Or, if you want to be more careful?:

    if (typeof options.MemberExpression === 'undefined') {